README.md

```diff
@@ -246,6 +246,7 @@ _See also: [WeblogMatrix](https://www.weblogmatrix.org/)_
 - [htmly](https://www.htmly.com/) - Databaseless Blogging Platform (Flat-File Blog). ([Demo](https://demo.htmly.com/), [Source Code](https://github.com/danpros/htmly)) `GPL-2.0` `PHP`
 - [Known](https://withknown.com/) - A collaborative social publishing platform. ([Source Code](https://github.com/idno/known)) `Apache-2.0` `PHP`
 - [PluXml](https://pluxml.org) - XML-based blog/CMS platform. ([Source Code](https://github.com/pluxml/PluXml)) `GPL-3.0` `PHP`
+- [Servus](https://github.com/ibz/servus) - a simple CMS / blog engine and personal Nostr relay that is fully self-contained within one executable file `GPL-3.0` `Rust`
```

Comment:
```suggestion
- [Servus](https://github.com/ibz/servus) - Simple CMS / blog engine and personal Nostr relay that is fully self-contained within one executable file. `GPL-3.0` `Rust`
```

- My main suggestion would be to add an installation heading or something similiar, this will make it easier to find out how to install it and make the read me more readable.